### ISSUE TITLE
[BUG] `--include latest-tag`  from CLI should take precedence over `exclude` of config file

### ISSUE THREAD

**Issue by felipecrs** [stackrox/kube-linter#456]
The docs says that:

> exclude always takes precedence, if you include and exclude the same check, KubeLinter always skips the check.

Which is ok. However, there should be an exception for when a check is excluded in a config file but included in the CLI. This means the user wants to force it.

My use case is simple: locally, developers should not be warned of `latest-tag` because I have a CI job that should ideally change and fix the charts. However, in CI, I want to still use the same config, but enabling back the `latest-tag` check to ensure the CI did its job of fixing the tags correctly.

**Comment by felipecrs** [comment]
AS a counter measure I do this as part of the CI script:

```bash
grep -v '  - latest-tag' .kube-linter.yaml >.kube-linter.yaml.tmp
mv -f .kube-linter.yaml.tmp .kube-linter.yaml
```

Which is of course not ideal.
